backend/test/followers_institution_handler_test.py

```diff
@@ -20,165 +19,105 @@ def setUp(cls):
-                        "Mayze should be in institution followers")
-        self.assertTrue(self.certbio.key in self.mayza.follows,
-                        "SpLab should be in user follows")
-
-        # Call the post method again
-        self.testapp.post("/api/institutions/%s/followers" %
-                          self.certbio.key.urlsafe())
-        # Confirmed that follow only one time
-        self.assertEquals(len(self.certbio.followers), 1,
-                          "The number of followers expected was 1")
-        self.assertEquals(len(self.mayza.follows), 1,
-                          "The number of follows expected was 1")
-
-    @patch('utils.verify_token', return_value={'email': 'mayzabeel@gmail.com'})
-    def test_delete(self, verify_token):
-        """Test the institution_follower_handler delete method."""
-        # Verified objects
-        self.assertEquals(len(self.certbio.followers), 0,
-                          "The number of followers expected was 0")
-        self.assertEquals(len(self.mayza.follows), 0,
-                          "The number of follows expected was 0")
+        user = user.key.get()
+        institution = institution.key.get()
+
+        # Assert that the user follows the institution
+        self.assertEquals(len(user.follows), 1,
+                          "The user should follow institution")
+        self.assertEquals(len(institution.followers), 1,
+                          "The institution should have a follower")
+        self.assertTrue(user.key in institution.followers,
+                        "user should be an institution's follower")
+        self.assertTrue(institution.key in user.follows,
+                        "institution should be in user.follows")
 
         # Call the post method
         self.testapp.post("/api/institutions/%s/followers" %
-                          self.certbio.key.urlsafe())
-
-        # Update the objects
-        self.mayza = self.mayza.key.get()
-        self.certbio = self.certbio.key.get()
-
-        # Verified objects
-        self.assertEquals(len(self.certbio.followers), 1,
-                          "The number of followers expected was 1")
-        self.assertEquals(len(self.mayza.follows), 1,
-                          "The number of follows expected was 1")
+                          institution.key.urlsafe())
+        # Assert that the user hasn't been added to the institution's followers
+        # again
+        self.assertEquals(len(institution.followers), 1,
+                          "The institution should have a follower")
+        self.assertEquals(len(user.follows), 1,
+                          "The user should follow institution")
+
+    @patch('utils.verify_token', return_value={'email': 'user@email.com'})
+    def test_delete(self, verify_token):
+        """Test the institution_follower_handler delete method."""
+        user = mocks.create_user()
+        user.email = ['user@email.com']
```

Comment:
We can improve the user creation by adding an optional argument "email", so we don't to set the email user after creating it.